### ISSUE TITLE
Flake8 does not support typing.no_type_check [REPLACEMENT ISSUE]

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#781]
The original issue

	Id: 697
	Title: Flake8 does not support typing.no_type_check

could not be created.
This is a dummy issue, replacing the original one. It contains everything but the original issue description. In case the gitlab repository is still existing, visit the following link to show the original issue:

TODO

**Comment by asottile** [comment]
In GitLab by @liiight on Nov 28, 2020, 08:41

changed the description

**Comment by asottile** [comment]
In GitLab by @asottile on Nov 28, 2020, 09:15

please follow the issue template, notably:

```
<!--
*************************************************************************
NOTE: flake8 is a linting framework and does not implement any checks

if you are reporting a problem with a particular check, please track down
the plugin which implements that check.

some common ones:
- F###: https://github.com/pycqa/pyflakes
- E###, W###: https://github.com/pycqa/pycodestyle
*************************************************************************
-->
```

**Comment by asottile** [comment]
In GitLab by @liiight on Nov 28, 2020, 10:11

Thanks, I apologize for opening this here

**Comment by asottile** [comment]
In GitLab by @asottile on Nov 28, 2020, 10:15

an aside, I'd be very careful about using libraries which override type annotations to have custom behaviours -- they're all likely to break in python3.10 (or with `from __future__ import annotations`)

**Comment by asottile** [comment]
In GitLab by @liiight on Nov 28, 2020, 10:18

hmm, good tip. I'll point the library author to it. 
Thanks!

**Comment by asottile** [comment]
In GitLab by @asottile on Nov 28, 2020, 10:20

you may also want to suggest [PEP 593](https://www.python.org/dev/peps/pep-0593/)

**Comment by asottile** [comment]
In GitLab by @liiight on Nov 28, 2020, 10:32

Thanks again. 
I opened https://github.com/PyCQA/pyflakes/issues/595, hopefully in the correct place this time.

I just want to point out that the wording the discenrs between flake8 and the checks that uses it could maybe be simplified or worded differently. 
Something like:
```
NOTE: flake8 is a linting framework and does not implement any checks

Checks occur via plugins to the flake8 framework, and are not part of it's core.
If you are reporting a problem with a particular check, please track down
the plugin which implements that check. You can do that via its error code prefix letter.

Some common ones:
- F###: https://github.com/pycqa/pyflakes
- E###, W###: https://github.com/pycqa/pycodestyle
```
